tools/docker/docker-compose-dev.yaml

```diff
@@ -269,6 +269,8 @@ services:
       interval: 30s
       timeout: 5s
       retries: 10
+    volumes:
```

Comment:
Does this work require this change?

Reply:
The prior solution was developed by @chrismeyersfsu and would re-build the image with a new settings file put inside of the source code. This didn't work for me any longer with some of the newer settings in DAB, which would aggressively error when the default wasn't in place (for reverse-sync). Along with the new image, the volume is needed so that local changes to the source tree will be reflected in all containers. Otherwise DAB will error on import due to the "service path" setting not being defined.